apps/docs/scripts/functions/fetchApiSource.ts

```diff
@@ -4,7 +4,7 @@ import path from 'path'
 import { TLDRAW_PACKAGES_TO_INCLUDE_IN_DOCS } from './package-list'
 
 const octokit = new Octokit({
-	auth: process.env.GITHUB_ACCESS_TOKEN,
+	auth: process.env.ACCESS_TOKEN,
```

Comment:
Not sure when this changed, but the environment variable is named `ACCESS_TOKEN`, not `GITHUB_ACCESS_TOKEN`.